tensorflow_lite_support/python/test/task/vision/image_classifier_test.py

```diff
@@ -128,6 +128,39 @@ def test_classify_model(self, model_file_type, max_results,
     self.assertDeepAlmostEqual(
         image_result_dict, expected_result_dict, places=5)
 
+  def test_equal(self):
+    base_options1 = _BaseOptions(
+        model_file=_ExternalFile(file_name=self.model_path))
+    options1 = _ImageClassifierOptions(base_options=base_options1)
+    classifier1 = _ImageClassifier.create_from_options(options1)
+    # Checks the same classifier object.
+    self.assertEqual(classifier1, classifier1)
+
+    base_options2 = _BaseOptions(
+      model_file=_ExternalFile(file_name=self.model_path))
+    options2 = _ImageClassifierOptions(base_options=base_options2)
+    classifier2 = _ImageClassifier.create_from_options(options2)
+    # Checks the classifiers with same file name.
+    self.assertEqual(classifier2, classifier2)
```

Comment:
classifier1

Reply:
@ziyeqinghan Thanks for pointing it out! Fixed it